examples/planning/README.md

```diff
@@ -0,0 +1,57 @@
+# Introduction to Planning for Self Driving Vehicles
+Planning is an essential piece of the Autonomous Vehicles (AV) stack.
+
+In this series of notebooks you will train and evaluate a data-driven ML policy.
+
+**Before starting, please download the [Lyft L5 Prediction Dataset 2020](https://self-driving.lyft.com/level5/prediction/) and follow [the instructions](https://github.com/lyft/l5kit#download-the-datasets) to correctly organise it.**
+
+Training an effective ML policy for planning can takes several hours on the most performing hardware.
+For this reason, we provide trained models you can experiment with in our evaluations notebooks,
+without requiring to train one yourself. Still, we suggest you to go through the training notebook we provide,
+as it contains key insights on how to effectively train a ML policy for planning.
+
+# [Training Notebook](./train.ipynb)
+
+In this notebook you are going to train your own ML policy to fully control a SVD. You will train your model using the Lyft Prediction Dataset and L5Kit
```

Comment:
```suggestion
In this notebook you are going to train your own ML policy to fully control a SDV. You will train your model using the Lyft Prediction Dataset and L5Kit
```
Same mistake in the figure
![image](https://user-images.githubusercontent.com/62287299/100733538-3fb06c80-33c6-11eb-9c86-d34e00ba789d.png)

Reply:
LOL